cmd/weaviate/README.md

```diff
@@ -252,3 +263,60 @@ with weaviate.connect_to_local(
 ```
 
 ### BM25 search (TODO)
+
+## Async Tenant Download
+
+When you freeze a tenant which is registered with the querier node, the querier node will
```

Comment:
Q: Will it work for already frozen tenants that got frozen without metadata server?

Say FROZEN (without metdata server) -> ACTIVE -> ADD more objects -> FROZEN -> Query.

Reply:
Trying to think, if we want to enable async download for already existing frozen customers on prod envs.

Reply:
I don't think we should automatically async download all frozen tenants as we don't currently have any indication which of them will want to be read. However, I think we may want to handle "async download for already existing frozen customers" (eg prefetching) in the future. We could do this by sending sending events or just by submitting simple fake read queries for them